openstack/openstack_performance/values.yaml

```diff
@@ -0,0 +1,13 @@
+# Default values for locust.
+# This is a YAML-formatted file.
+locustCmd: "locust -f /data/src/openstack_performance/openstack.py OpenstackUser"
```

Comment:
what is the point for duplicating this in both entrypoint and in args of the deployment?